src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/AttributeDirective.ts

```diff
@@ -13,6 +13,10 @@ export function RunAttributeDirectiveSuite() {
             await assertMatchesSnapshot('@attribute');
         });
 
+        it('As C# local', async () => {
```

Comment:
@JoeRobich if the C# grammar tests could be more like this (snapshot based) it'd make contributing significantly easier.